engine/src/test/scala/cromwell/webservice/MetadataBuilderActorSpec.scala

```diff
@@ -292,15 +291,16 @@ class MetadataBuilderActorSpec extends TestKitSuite("Metadata") with FlatSpecLik
     val kisv3 = ("key[0]:subkey", "value3", OffsetDateTime.now.plusSeconds(2))
     val kiv4 = ("key[0]", "value4", OffsetDateTime.now.plusSeconds(3))
 
-    val t = Table(
-      ("list", "res"),
+    val t = List(
       (List(kv),  """"key": "value""""),
       (List(kv, ksv2),  """"key": { "subkey": "value2" }"""),
       (List(kv, ksv2, kisv3),  """"key": [ { "subkey": "value3" } ]"""),
       (List(kv, ksv2, kisv3, kiv4),  """"key": [ "value4" ]""")
     )
 
-    forAll(t) { (l, r) => assertMetadataKeyStructure(l, r) }
+    Future.sequence(t map { case (l, r) => assertMetadataKeyStructure(l, r) }) map { assertions =>
```

Comment:
It can be done, not sure it's worth it though:

```
  def assertMetadataKeyStructure(eventList: List[EventBuilder],
                                 expectedJson: String) = {
    assertMetadataKeyStructure(eventList, expectedJson, WorkflowId.randomId(), makeEvent)
  }
.
.
.
    import cats.instances.list._
    import cats.instances.future._
    import cats.syntax.traverse._

    val assert_ = (assertMetadataKeyStructure _).tupled

    t traverse assert_ map { assertions =>
      assertions should contain only Succeeded
    }
```